sktime/classification/distance_based/_time_series_neighbors.py

```diff
@@ -499,6 +499,25 @@ def predict_proba(self, X):
         return probabilities
 
 
+# Helper function to convert univariate data into a 2D numpy array
+# or multivariate data into a 3D numpy array
+def convert_data(X):
```

Comment:
Note that we have [`tabularize`](https://github.com/alan-turing-institute/sktime/blob/7946e3588d0e526193be96ebd5ca250f07fb8890/sktime/utils/data_container.py#L57) which may be of use here, otherwise I'd suggest moving this function into `sktime/utils/data_container` but would prefer to use or extend existing functionality